templates/node_inspector.html

```diff
@@ -45,20 +60,22 @@
         {% include 'node_inspector_graph_comment.html' %}
         <!-- /ko -->
 
-        <div class="row mt-1">
+        {% include 'node_inspector_hidden_bs_trigger.html' %}
```

Comment:
In my branch (eagle-305-2), I was able to remove the need for this hidden trigger.

It's a little complicated to explain how here in text, but hopefully it is obvious if you take a look at the code. Otherwise we can discuss it on Thursday.